src/Microsoft.Health.Fhir.Shared.Web/Startup.cs

```diff
@@ -121,6 +121,7 @@ private void AddDataStore(IServiceCollection services, IFhirServerBuilder fhirSe
             if (runtimeConfiguration is AzureApiForFhirRuntimeConfiguration)
             {
                 fhirServerBuilder.AddCosmosDb();
+                fhirServerBuilder.AddCosmosdbMI();
```

Comment:
Why not always register it, there is a separate config setting controlling this also

Reply:
Updated!